src/main/java/EntryPoint/config/SecurityConfig.java

```diff
@@ -0,0 +1,40 @@
+package EntryPoint.config;
+
+import EntryPoint.filter.AuthenticationFilter;
+import EntryPoint.utils.JwtUtil;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configurers.AbstractHttpConfigurer;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.web.SecurityFilterChain;
+
+@Configuration
+@EnableWebSecurity
+public class SecurityConfig {
+
+    private final JwtUtil jwtUtil;
+
+    public SecurityConfig(JwtUtil jwtUtil) {
+        this.jwtUtil = jwtUtil;
+    }
+
+    @Bean
+    public BCryptPasswordEncoder passwordEncoder() {
+        return new BCryptPasswordEncoder();
+    }
+
+    @Bean
+    public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
+        http
+                .csrf(AbstractHttpConfigurer::disable) // Disable CSRF for APIs
```

Comment:
use disable function. like disable()

Reply:
I am using spring security 6.x+, that's why this style of CSRF disable is correct. For previous version, http.csrf().disable() is compatible, but not for that.

Reply:
If this is not compatible then .csrf(AbstractHttpConfigurer::disable) is okay.